torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -1226,6 +1228,60 @@ class IrParser {
       }
     }
 
+    {
+      auto ptr_op = getOperatorForLiteral(
+          "aten::autocast_to_fp16(Tensor(a) self) -> Tensor(a)");
+      registerParseRule(
+          ptr_op,
+          [](const Node* node,
+             std::unordered_map<size_t, CgValue>& value_map) -> void {
+            auto self = value_map[node->input()->unique()];
+            auto out = unaryOp(UnaryOpType::Set, self);
+            value_map.emplace(node->output()->unique(), out);
+          });
+    }
+
+    {
+      auto ptr_op = getOperatorForLiteral(
+          "aten::autocast_to_fp32(Tensor(a) self) -> Tensor(a)");
+      registerParseRule(
+          ptr_op,
+          [](const Node* node,
+             std::unordered_map<size_t, CgValue>& value_map) -> void {
+            auto self = value_map[node->input()->unique()];
+            auto out = unaryOp(UnaryOpType::Set, self);
+            value_map.emplace(node->output()->unique(), out);
+          });
+    }
+
+    // Limiting aten::to implementation to only change the dtype of a tensor
+    {
+      auto ptr_op = getOperatorForLiteral(
+          "aten::to.dtype(Tensor self, ScalarType dtype, bool non_blocking=False, bool copy=False, MemoryFormat? memory_format=None) -> Tensor");
+      registerParseRule(
+          ptr_op,
+          [](const Node* node,
+             std::unordered_map<size_t, CgValue>& value_map) -> void {
+            const auto self = value_map[node->input(0)->unique()];
+            auto dtype = toIValue(node->input(1))->toScalarType();
+
+            for (size_t i = 2; i < 5; ++i) {
```

Comment:
maybe relax this a little bit since we are not using them.

Reply:
AFAIK, we are not respecting `copy` nor `non_blocking`, so just drop them would be fine.